install_sedumi.m

```diff
@@ -156,6 +163,16 @@ function install_sedumi( varargin )
     IS64BIT  = ~ISOCTAVE & strcmp(COMPUTER(end-1:end),'64');
     flags = {};
     libs = {};
+    
+    if ~isempty(openblas_path)
+      if ~isdir(openblas_path)
+        disp('OpenBLAS path not valid!');
```

Comment:
Maybe be a bit more helpful to the user?

disp( 'Please set the OpenBLAS include path with     install_sedumi -obpath=/path/to/openblas/headers' )